src/main/java/eu/nimble/service/delegate/Delegate.java

```diff
@@ -401,8 +442,300 @@ public Response postPartySearch(Map<String, Object> body) {
     }
     
     /***********************************   indexing-service/party/search - END   ***********************************/
-    
-    
+
+    /***********************************   Catalogue Service **************************************************/
+
+    /***********************************   /catalogue/{catalogueUuid}/catalogueline/{lineId} ***************************************/
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/catalogue/{catalogueUuid}/catalogueline/{lineId}")
+    public Response federatedGetCatalogueLine(@Context HttpHeaders headers, @PathParam("catalogueUuid") String catalogueUuid, @PathParam("lineId") String lineId, @QueryParam("delegateId") String delegateId) {
```

Comment:
need to remove path param delegateId, the api has to be identical to catalog service